samples/stock/stocktools/scrollable-popup/demo.js

```diff
@@ -0,0 +1,31 @@
+Highcharts.getJSON('https://www.highcharts.com/samples/data/aapl-ohlcv.json', function (data) {
+    var ohlc = [],
+        dataLength = data.length,
+        i = 0;
+    for (i; i < dataLength; i += 1) {
+        ohlc.push([
+            data[i][0], // the date
+            data[i][1], // open
+            data[i][2], // high
+            data[i][3], // low
+            data[i][4] // close
+        ]);
+    }
```

Comment:
I think we could use simple line series with few points - no need to load `data.js` module, fetch data from server and iterate through the whole array. We could also remove 3 lines in the series' config (type, id, name).